### PR TITLE
bpo-4630: Fix errors in Lib/idlelib/NEWS.txt

### PR DESCRIPTION
**Pull request by ZackerySpytz**
https://bugs.python.org/issue4630

### CODE REVIEW

File: Lib/idlelib/NEWS.txt [hunk 1]
```diff
@@ -63,9 +63,9 @@ bpo-38636: Fix IDLE Format menu tab toggle and file indent width. These
 functions (default shortcuts Alt-T and Alt-U) were mistakenly disabled
 in 3.7.5 and 3.8.0.
 
-bpo-4360: Add an option to toggle IDLE's cursor blink for shell,
+bpo-4630: Add an option to toggle IDLE's cursor blink for shell,
```
Comment (terryjreedy) [inline Lib/idlelib/NEWS.txt:66]: This was a typo

File: Lib/idlelib/NEWS.txt [hunk 2]
```diff
@@ -63,9 +63,9 @@ bpo-38636: Fix IDLE Format menu tab toggle and file indent width. These
...
 editor, and output windows.  See Settings, General, Window Preferences,
-Cursor Blink.  Patch by Zachary Spytz.
+Cursor Blink.  Patch by Zackery Spytz.
```
Comment (terryjreedy) [inline Lib/idlelib/NEWS.txt:68]: but this is an error, for which I apologize.

**Review by terryjreedy — APPROVED** [review]
I expect the backport to 3.9 will work.  I believe the NEWS.txt change would backport to 3.8 would work, but the Misc/NEWS.d to not do so.  If is does not, you can submit a separate [3.8] ... PR.

**Comment by terryjreedy** [comment]
@Mariatta , @vstinner  After 7 hours, travis has not reported, and there is no Details link to halt or restart. 
(Will close and reopen to restart all)

**Comment by vstinner** [comment]
> @Mariatta , @vstinner After 7 hours, Travis has not reported, and there is no Details link to halt or restart

Yeah, it happens: see https://github.com/python/core-workflow/issues/371

**Pull request merged**